firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -301,6 +329,72 @@ public Query whereArrayContains(@NonNull FieldPath fieldPath, @NonNull Object va
     return whereHelper(fieldPath, Operator.ARRAY_CONTAINS, value);
   }
 
+  /**
+   * Creates and returns a new Query with the additional filter that documents must contain the
+   * specified field, the value must be an array, and that the array must contain any values of the
+   * provided array.
+   *
+   * <p>A Query can have only one whereArrayContainsAny() filter.
+   *
+   * @param field The name of the field containing an array to search
+   * @param value The array that contains the values to match.
+   * @return The created Query.
+   */
+  @NonNull
+  @PublicApi
+  public Query whereArrayContainsAny(@NonNull String field, @NonNull List<Object> value) {
+    return whereHelper(FieldPath.fromDotSeparatedPath(field), Operator.ARRAY_CONTAINS_ANY, value);
+  }
+
+  /**
+   * Creates and returns a new Query with the additional filter that documents must contain the
+   * specified field, the value must be an array, and that the array must contain any values of the
+   * provided array.
+   *
+   * <p>A Query can have only one whereArrayContainsAny() filter.
+   *
+   * @param fieldPath The path of the field containing an array to search
+   * @param value The array that contains the values to match.
+   * @return The created Query.
+   */
+  @NonNull
+  @PublicApi
+  public Query whereArrayContainsAny(@NonNull FieldPath fieldPath, @NonNull List<Object> value) {
+    return whereHelper(fieldPath, Operator.ARRAY_CONTAINS_ANY, value);
+  }
+
+  /**
+   * Creates and returns a new Query with the additional filter that documents must contain the
+   * specified field and that the field must equal any values of the provided array.
```

Comment:
I'd maybe say, "must contain the specified field and the value must equal one of the values from the provided array."